src/samplers/gamma.jl

```diff
@@ -100,8 +93,8 @@ function rand(s::GammaGDSampler)
 
     # Step 8
     @label step8
```

Comment:
should we maybe  replace the label & goto macros with something a bit easier to read? (splitting functions, if else branches)

Reply:
These labels come from the original paper (and again, probably out of scope of the PR).

Reply:
I've replaced the gotos and added one function (to calculate q), but I can't replace the labels as I don't know what they mean...